### PR TITLE
[ACS][Chat] Added Meeting Chat Sample

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
Added new sample for meeting chat

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by jimchou-dev — APPROVED** [review]

**Comment by vhuseinova-msft** [comment]
Should we also add a CI check for this sample?

File: ui-library-quickstart-teams-interop-meeting-chat/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+- [Add file sharing with UI Library in Azure Communication Service Chat](./file-sharing-tutorial-acs-chat.md)
+- [Add file sharing with UI Library in Teams Interoperability Chat](./file-sharing-tutorial-interop-chat.md)
+
+And this sample is built based on the [CallWithChat Compsite](https://azure.github.io/communication-ui-library/?path=/docs/composites-call-with-chat-basicexample--basic-example) from our storybook page.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)  .
+- [Node.js](https://nodejs.org/en/) Active LTS and Maintenance LTS versions (8.11.1 and 10.14.1 recommended).
```
Comment (vhuseinova-msft) [inline ui-library-quickstart-teams-interop-meeting-chat/README.md:22]: Is it a correct version for node (as they look outdated)?

File: ui-library-quickstart-teams-interop-meeting-chat/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "ui-library-quickstart-teams-interop-meeting-chat",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/communication-react": "1.8.0-beta.1",
+    "@fluentui/react": "8.101.0",
+    "@testing-library/jest-dom": "^5.14.1",
```
Comment (vhuseinova-msft) [inline ui-library-quickstart-teams-interop-meeting-chat/package.json:8]: Do we want to include the testing dependencies?

File: ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  // The adapter is created asynchronously by the useAzureCommunicationCallWithChatAdapter hook.
+  // Here we show a spinner until the adapter has finished constructing.
+  if (!adapter) {
+    return <Spinner label="Initializing..." />;
+  }
+
+  return <CallWithChatComposite adapter={adapter} fluentTheme={props.fluentTheme} options={props.compositeOptions} />;
+};
+
+// initializeIcons();
```
Comment (vhuseinova-msft) [inline ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx:77]: Do we still need this line?

File: ui-library-quickstart-teams-interop-meeting-chat/README.md (hunk 1 above)
Comment (jpeng-ms) [reply ui-library-quickstart-teams-interop-meeting-chat/README.md]: thanks updated!

File: ui-library-quickstart-teams-interop-meeting-chat/package.json (hunk 2 above)
Comment (jpeng-ms) [reply ui-library-quickstart-teams-interop-meeting-chat/package.json]: good catch

File: ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx (hunk 3 above)
Comment (jpeng-ms) [reply ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx:77]: thanks removed

File: ui-library-quickstart-teams-interop-meeting-chat/package.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "ui-library-quickstart-teams-interop-meeting-chat",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/communication-react": "1.8.0-beta.1",
+    "@fluentui/react": "8.101.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "@types/jest": "^26.0.24",
```
Comment (vhuseinova-msft) [inline ui-library-quickstart-teams-interop-meeting-chat/package.json:10]: I think 
`"@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "@types/jest": "^26.0.24",`
    
    can be removed

File: ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx (hunk 3 above)
Comment (vhuseinova-msft) [reply ui-library-quickstart-teams-interop-meeting-chat/src/App.tsx:77]: It looks like it's not pushed

**Review by vhuseinova-msft — APPROVED** [review]
A couple of minor comments but otherwise looks good!

File: ui-library-quickstart-teams-interop-meeting-chat/package.json (hunk 4 above)
Comment (jpeng-ms) [reply ui-library-quickstart-teams-interop-meeting-chat/package.json:10]: right thank you!

**Pull request merged**